gitapp/lib/users_list/users_presenter.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:gitapp/users_list/api_repository.dart';
+import 'package:gitapp/users_list/user_model.dart';
+
+class UsersPresenter {
+  List<User> userList = List.empty(growable: true);
+  List<User> searchedList = List.empty(growable: true);
+
+  UsersPresenter();
+
+  Future<void> fetchUsersFromApi(String username) async {
```

Comment:
This name of method suggests that it should return user from APi, but it does not.
In this case Update would be a better solution

Reply:
Solved